### DISCUSSION TITLE
WSL2 networking

### DISCUSSION THREAD

**Discussion by privacyguy123** [microsoft/WSL#5249] [General]
First off, the new stuff announced for WSL2 is amazing - great work guys!

However, is it possible to bring back the old WSL networking in WSL2? I and many others found it much easier to work with ...

**Comment by billgatesfan** [comment]
"bring back the old WSL networking" : I bet it will be difficult :-)

Myself, I am looking for a way to do "bridging" in WSL2 instead of NAT, which is configured by default using the HyperV "WSL" vSwitch. Unfortunately, I already have another brigded/external vSwitch for my HyperV VMs, and it seems only one physical interface can be used for an external network vSwitch. Sad :-(.

Is it what you meant?
